libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -283,4 +300,54 @@ void ReferencesResolver::operator()(yul::VariableDeclaration const& _varDecl)
 		visit(*_varDecl.value);
 }
 
+void ReferencesResolver::resolveInheritDoc(StructuredDocumentation const& _documentation, StructurallyDocumentedAnnotation& _annotation)
+{
+	switch (_annotation.docTags.count("inheritdoc"))
+	{
+	case 0:
+		break;
+	case 1:
+	{
+		string const& name = _annotation.docTags.find("inheritdoc")->second.content;
+		vector<string> path;
+		boost::split(path, name, boost::is_any_of("."));
+		Declaration const* result = m_resolver.pathFromCurrentScope(path);
```

Comment:
Do you have a test case for that? You will need to use a multi-file test that uses import.

Reply:
This test does a resolving of a path: https://github.com/ethereum/solidity/pull/9218/files#diff-5dcd49ff3e08e5ff224ee08fbb7ac514

It's not a contract and thus fails later but it does successfully resolve the path.

Reply:
Can you please still add a multi-file test?